modules/serverless-logic-eventing-system-config-precedence-rule.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-workflow-eventing-system
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-eventing-system-config-precedence-rule_{context}"]
+= Eventing system configuration precedence rules
```

Comment:
"rules" is lowercase here and this was intended?

Reply:
The usage of "rules" here is correct. it was not intended.